decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -65,6 +65,14 @@ def order
         @order = params[:order] || "random"
       end
 
+      def order_fields
+        @order_fields ||= begin
```

Comment:
I think the `return` is much more clear than using a `begin` with no exception rescue, sincerely. :/